framework/test/unit/modules/interoperability/commands/channel_terminated.spec.ts

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
This file should be under `framework/test/unit/modules/interoperability/cc_commands`

Reply:
changed